src/app/components/navbar/navbar.html

```diff
@@ -5,11 +5,11 @@
         </div>
         <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-6">
             <ul class="nav navbar-nav">
-                <li><a ui-sref="home">Home</a></li>
+                <li><a ui-sref="home"></a></li>
```

Comment:
it does not have to be empty, if you put text such as "Home" to it, it will display the text. I personally do not like the text "home" next to the transmart icon, so I removed the text.